### ISSUE TITLE
How to use custom prefix color?

### ISSUE THREAD

**Issue by oblicue** [5zig-reborn/The-5zig-Mod#149]
I do not understand how to use the "Custom Prefix Color" option in Formatting Options. 

Could someone please provide a way to use this? I have tried inputting "255, 234, 141" into the input box, to get a custom colour, but minecraft crashes straight away and i have to open the config.json for 5zig and manually change the value back to nothing before minecraft will relaunch. If this is not implemented or if nobody knows how to explain its use, maybe an rgb slider could be implemented to change this?

TL;DR: How can I get a custom rgb using Custom Prefix Color

**Comment by oblicue** [comment]
by the way, this setting is in 5zigmod -> display settings -> formatting options -> custom prefix color
